safebrowser.go

```diff
@@ -358,6 +359,12 @@ func (sb *SafeBrowser) Status() (Stats, error) {
 	return stats, sb.db.Status()
 }
 
+// WaitUntilReady blocks until the database is not in an error state,
+// or until the provided Context is cancelled.
```

Comment:
Nit: Go style is canceled, which one L. It bothers me that this is the norm, but yay for consistency.